girder/utility/filesystem_assetstore_adapter.py

```diff
@@ -332,19 +332,39 @@ def _importDataAsItem(self, name, user, folder, path, files,
             self.importFile(item, os.path.join(path, fname),
                             user, name=fname)
 
+    def _hasOnlyFiles(self, path, files):
+        return all(map(os.path.isfile, (os.path.join(path, fname)
```

Comment:
nit: Since this is a comprehension in a map in an all, can we have a small `#` comment. :smile: